core/src/main/java/org/bitcoinj/utils/ListenableCompletableFuture.java

```diff
@@ -52,7 +59,9 @@ public static <T> ListenableCompletableFuture<T> completedFuture(T value) {
      * @param throwable the exceptions
      * @param <T> the type of the expected value
      * @return the completed CompletableFuture
+     * @deprecated Use {@link FutureUtils#failedFuture(Throwable)} or if JDK 9+ use {@code CompletableFuture.failedFuture}
```

Comment:
FutureUtils is an internal class, so it's probably not ok to recommend to use it?

Reply:
Good catch. I was very much in an "internal state of mind" both when I wrote these JavaDoc originally and when editing them.

Fixed and force-pushed.